io-ballerina/file_byte_io.bal

```diff
@@ -19,7 +19,7 @@
 # byte[]|io:Error content = io:fileReadBytes("./resources/myfile.txt");
```

Comment:
Since this is an anti-pattern, let's make use of `check`

Reply:
Here we need to indicate the error is returning and what type of error(`io:Error`) is going to return. Therefore, I think we should explicitly mention the error return rather than using the check expression.

Reply:
@shafreenAnfar, @daneshk WDYT

Reply:
As discussed let's keep it as it is.